packages/edit-site/src/components/global-styles/shadows-panel.js

```diff
@@ -115,6 +152,23 @@ function ShadowList( { label, shadows, category, canCreate, onCreate } ) {
 						/>
 					</FlexItem>
 				) }
+				{ !! shadows?.length && category === 'custom' && (
+					<Menu
+						trigger={
+							<Button
+								size="small"
+								icon={ moreVertical }
+								label={ __( 'shadow options' ) }
+							/>
+						}
+					>
+						<Menu.Item onClick={ onReset }>
+							<Menu.ItemLabel>
+								{ __( 'Remove all shadows' ) }
```

Comment:
```suggestion
								{ __( 'Remove all custom shadows' ) }
```

I think we should be more specific that this will remove just the custom shadows.